plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -40,50 +40,34 @@
 
     private IcebergQueryRunner() {}
 
-    public static DistributedQueryRunner createIcebergQueryRunner(Map<String, String> extraProperties, List<TpchTable<?>> tpchTables)
+    public static DistributedQueryRunner createIcebergQueryRunner()
             throws Exception
     {
-        FileFormat defaultFormat = new IcebergConfig().getFileFormat();
-        return createIcebergQueryRunner(extraProperties, defaultFormat, tpchTables);
-    }
-
-    public static DistributedQueryRunner createIcebergQueryRunner(Map<String, String> extraProperties)
-            throws Exception
-    {
-        FileFormat defaultFormat = new IcebergConfig().getFileFormat();
-        return createIcebergQueryRunner(extraProperties, defaultFormat, TpchTable.getTables());
-    }
-
-    public static DistributedQueryRunner createIcebergQueryRunner(Map<String, String> extraProperties, FileFormat format)
-            throws Exception
-    {
-        return createIcebergQueryRunner(extraProperties, format, TpchTable.getTables());
-    }
-
-    public static DistributedQueryRunner createIcebergQueryRunner(Map<String, String> extraProperties, FileFormat format, List<TpchTable<?>> tables)
-            throws Exception
-    {
-        return createIcebergQueryRunner(extraProperties, format, tables, Optional.empty());
+        return createIcebergQueryRunner(
```

Comment:
nit: using one line for all args seems okay

Reply:
i decided to lay this out in lines, so it's easier to update in the future.
this doesn't affected readability, since it's just a pass-thru.